boards/samr21-xpro/include/board.h

```diff
@@ -73,12 +73,34 @@ extern "C" {
 #define BTN0_PIN            GPIO_PIN(0, 28)
 #define BTN0_MODE           GPIO_IN_PU
 /** @} */
+/**
+ * @name    RFCTL pin definitions and handlers
+ * @{
+ */
+#define RFCTL1_PIN            GPIO_PIN(0, 9)
+#define RFCTL2_PIN            GPIO_PIN(0, 12)
+
+#define RFCTL_PORT            PORT->Group[0]
+#define RFCTL1_MASK           (1 << 9)
+#define RFCTL2_MASK           (1 << 12)
+
+#define RFCTL1_ON             (RFCTL_PORT.OUTCLR.reg = RFCTL1_MASK)
+#define RFCTL1_OFF            (RFCTL_PORT.OUTSET.reg = RFCTL1_MASK)
+#define RFCTL1_TOGGLE         (RFCTL_PORT.OUTTGL.reg = RFCTL1_MASK)
```

Comment:
Why toggle here and below? Shouldn't we sure which antenna we want to choose?

Reply:
I just took the code from the LED and changed the names and did not spent any time thinking about the use of the toggle define.
By now there is no use for it.